src/xasl/access_json_table.cpp

```diff
@@ -234,6 +234,22 @@ namespace cubxasl
       return error_code;
     }
 
+    void
+    column::clear_xasl (bool is_final_clear /* = true */)
+    {
+      if (is_final_clear)
+	{
+	  (void) pr_clear_value (m_on_empty.m_default_value);
+	  (void) pr_clear_value (m_on_error.m_default_value);
+	}
+
+      (void) pr_clear_value (m_output_value_pointer);
```

Comment:
Do you want
```c
  if (m_output_value_pointer != NULL)
    {
      (void) pr_clear_value (m_output_value_pointer);
      (void) db_make_null (m_output_value_pointer);
    }
```

Reply:
The check is only mandatory for db_make_null, pr_clear_value does an early out if argument is null. However, I followed your suggestion for clarity.